packetbeat/main.go

```diff
@@ -31,6 +34,12 @@ func main() {
 	// })
 	// autopprof.Capture(autopprof.HeapProfile{})
 
+	go func(){
```

Comment:
At least, pass the address from a argument from command line or load from it environment variable. There will be cases where the port 8000 in the deployment machine is already bind.

Reply:
It's just for test, I will delete it after completion

Reply:
No...please don't do it. You should always test the production code.